doc/source/conf.py

```diff
@@ -107,7 +107,7 @@ def setup(app):
 
 # General information about the project.
 project = "GeoPandas"
-copyright = "2013–2022, GeoPandas developers"
+copyright = "2013–2024, GeoPandas developers"
```

Comment:
```suggestion
copyright = "2013–, GeoPandas developers"
```
This should solve it once and for all :)

Reply:
Or something like `f"2013-{datetime.date.today().year}"` if we want to keep the year but automate it?

Reply:
Fine with either.

Reply:
I think drop the year, that way there's no chance we get future issues/ pull requests indicating it's wrong (whereas we might if the generated docs from the last release have an old year).

As far as I understand it's not required anyway - https://hynek.me/til/copyright-years/